features/facebookImport/src/model/analyses/ministories/about-pictures-data-analysis.js

```diff
@@ -43,12 +47,17 @@ export default class AboutPicturesDataAnalysis extends RootAnalysis {
      * simply include all pictures in that folder.
      */
     async _userPicturesFromExport(zipFile) {
+        const photoRegexes = [
+            /^photos_and_videos\/(.+)\.(jpg|jpeg)$/,
```

Comment:
The extension is never going to be in caps?

Reply:
Only saw lowercase names up to now, but can easily make the regexes case insensitive.